mod/groups/start.php

```diff
@@ -329,7 +329,7 @@ function groups_set_icon_url($hook, $type, $url, $params) {
 		return "groupicon/$group->guid/$size/$icontime.jpg";
 	}
 
-	return "mod/groups/graphics/default{$size}.gif";
```

Comment:
I suspect the number of plugins pointing to `mod/plugin/someimage.jpg` is pretty high... No way that I know to forward with redirects. Really like pushing everything into views but this might be a case for views.php -- all gain no pain as far as I can tell...